pool.go

```diff
@@ -273,6 +273,13 @@ func (p *Pool) IsClosed() bool {
 
 // Release closes this pool and releases the worker queue.
 func (p *Pool) Release() {
+	if p.stopPurge != nil {
```

Comment:
Put this new code under line `if !atomic.CompareAndSwapInt32(&p.state, OPENED, CLOSED) {`.